src/codegeneration/TempVarTransformer.js

```diff
@@ -110,13 +113,15 @@ class VarScope {
 export class TempVarTransformer extends ParseTreeTransformer {
   /**
    * @param {UniqueIdentifierGenerator} identifierGenerator
+   * @param {Options} options
    */
-  constructor(identifierGenerator) {
+  constructor(identifierGenerator, options) {
```

Comment:
I contemplated doing this. My opinion was that it is cleaner to only pass what is needed but after doing all this work I think I've changed my mind. Let me make ParseTreeTransformer take three params and expose them as properties.